package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "5.0.50-sscs-sit-release",
+  "version": "5.0.49-restricted-case-access-v2",
```

Comment:
toolkit version has gone backwards - is this definitely what you want

Reply:
I have updated the version.